tests/general/InvalidLinkBearTest.py

```diff
@@ -126,6 +126,7 @@ def test_run(self):
         http://httpbin.org/status/410
         http://httpbin.org/status/500
         http://httpbin.org/status/503
+        http://httpbin.org/status/200 http://httpbin.org/status/404
```

Comment:
please remove this line. If multi-line breaks, this test method should pass, and `test_multiple_results_per_line` should fail.